source/braille.py

```diff
@@ -808,6 +820,13 @@ def getFormatFieldBraille(field, isAtStart, formatConfig):
 				# Translators: Displayed in braille for a heading with a level.
 				# %s is replaced with the level.
 				textList.append(_("h%s")%headingLevel)
+	if formatConfig["reportLinks"]:
+		link=field.get("link")
+		oldLink=fieldCache.get("link")
+		if link and link != oldLink:
+			textList.append(roleLabels[controlTypes.ROLE_LINK])
+	fieldCache.clear()

```

Comment:
> @jcsteh How is this for Python gotchas?

All good. The empty dict is created inside the function, so it's local to each run of that function. The gotcha only applies for keyword arguments. Think of keyword argument defaults as being at global scope, even though you can't access them outside of the function.